organizations.md

```diff
@@ -0,0 +1,115 @@
+|contact      |Contact(#contact)  |A contact hash representing how one should get in contact with the organization.
+
+_[Back to top...](#)_
+
+### Related Objects
+
+These JSON hashes included in the table above are broken out into their own tables for readability, rather than independent resources with their own endpoints.
+
+#### Contact
+
+|Name          |Type      |Description
+|-----------    |-----------|--------------
+|contact.name	|string	|Name of the host or contact person for organization (e.g., Jane Doe)
+|contact.email_address	|string	|Email address of the organization (jane.doe@hotmail.co.uk)
+|contact.phone_number	|string	|Phone number of the organization (214-555-0869)
+|contact.additional_info |string  |Free form place for information about how to contact the organization
+|contact.public|boolean |Whether the organization's info should be shared publicly (if false, should only be shared with members)
+
+#### Location
+
+|Name          |Type      |Description
+|-----------    |-----------|--------------
+|location.venue	|string	|Optional venue name at the event address, useful for names of buildings. (ex: Smith Hall)
+|location.address_lines	|strings[]	|An array of strings representing the event's street address.
+|location.locality	|string	|A city or other local administrative area.
+|location.region	|string	|State or subdivision codes according to ISO 3166-2 (Final 2 alpha digits).
+|location.postal_code	|string	|The region specific postal code, such as a zip code.
+|location.country	|string	|The country code according to ISO 3166-1 Alpha-2.
+|location.language	|string	|Language in which the address is recorded -- language code according to ISO 639.
+|location.location	|object	|An object hash representing the geocoded location information for the address.
+|location.location.latitude	|float	|A positive or negative float number representing the latitude of the address.
+|location.location.longitude	|float	|A positive or negative float number representing the longitude of the address.
+|location.location.accuracy	|enum	|A value representing the accuracy of the geocode. One of "Rooftop" or "Approximate".
+|location.public |boolean |Whether the venue's location should be shared publicly, or if false, only shared with RSVPs (for example, someone's house)
+
+### Links
+
+{% include links_intro.md %}
+
+|Name          	|Type		|Description
+|-----------    |-----------|--------------
+|self			|[Organization*](organizations.html)	|A self-referential link to the organization.
+|events   |[Events*](events.html)               |A link to the collection of events associated with the organization.
+|administrators |[Memberships*](memberships.html) |A link to the collection of membership resources that link an organization to its administrators.
+|memberships |[Memberships*](memberships.html)  |A link to the collection of memberships resources linking an organization to its members.
```

Comment:
I don't think these get two collections as you've written it -- would be one with a different type (which I think is the correct way), so only one link really.

Reply:
I wrote the Memberships resource as as one with a different type – I was viewing administrators as a link to a filtered collection of memberships (in case Memberships are in the thousands)

Reply:
Generally that's not how we've done filters -- they are not separate links, but rather defined over on the oData side of things: http://opensupporter.github.io/osdi-docs/#filtering-collections

I think the type of filtering you want here is implied by what we have already, you'd do administrator eq ‘true’ if the server supports it, but it wouldn't be in the links collection.